### PR TITLE
Windows version of bat

### PR DESCRIPTION
**Pull request by anykao**
I'd like to execute bat in powershell.

### CODE REVIEW

**Comment by anykao** [comment]
Note, the auth_token should change, and change email notification as you like.

**Comment by sharkdp** [comment]
I have enabled AppVeyor for this repository, so we should be able to trigger a build with the next commit on your branch.

Can you remind me how to set up the auth_token? Is it the same that we have in `.travis.yml`?

Is there anything else that I need to do?

I guess you have seen #210?

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    RUSTUP_HOME: "c:\\rustup"
+
+install:
+  - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
+  - rustup-init.exe -y --default-host %TARGET%
+  - set PATH=%PATH%;C:\cargo\bin
+  - rustc -Vv
+  - cargo -V
+  
+build: false
```
Comment (sharkdp) [inline appveyor.yml]: Why is this set to `false`? It currently doesn't build `bat` or run the tests. If we want automatic deployment for Windows, we definitely need to enable continuous integration.

**Comment by anykao** [comment]
Thank you for reviewing PR.
I have seen #210 , but as you said, there are some tests failed in windows.
image
Should we anyway add `cargo test` to Windows build.

As for auth_token. The same method as `fd` will be suffice, that is 
```
  # Here's how:
  # - Go to 'https://github.com/settings/tokens/new' and generate a Token with only the
  # `public_repo` scope enabled
  # - Then go to 'https://ci.appveyor.com/tools/encrypt' and enter the newly generated token.
  # - Enter the "encrypted value" below
```

**Comment by anykao** [comment]
Add cargo test and disabled test_snapshots on windows.
If there are any other changes should be done, just let me know.

**Comment by sharkdp** [comment]
I have split the style-tests into multiple tests to see which of them fail.

**Comment by sharkdp** [comment]
I found out why the Git modifications were not shown on Windows. Now all unit tests pass :sparkles:

**Pull request merged**